arbnode/batch_poster.go

```diff
@@ -1334,6 +1385,26 @@ func (b *BatchPoster) maybePostSequencerBatch(ctx context.Context) (bool, error)
 		}
 	}
 
+	delayBuffer, err := GetDelayBufferConfig(ctx, b.seqInbox)
+	if err != nil {
+		return false, err
+	}
+	if delayBuffer.Enabled && b.building.firstDelayedMsg != nil {
+		latestHeader, err := b.l1Reader.LastHeader(ctx)
+		if err != nil {
+			return false, err
+		}
+		latestBlock := latestHeader.Number.Uint64()
+		firstDelayedMsgBlock := b.building.firstDelayedMsg.Message.Header.BlockNumber
+		if latestBlock >= firstDelayedMsgBlock+delayBuffer.Threshold {
```

Comment:
I agree; I will add an option. When writing the code, I considered Arb1 and ArbNova, where the delay buffer will almost always not trigger. And if it triggers, it will most likely replenish the buffer. However, this might indeed be an issue for low-activity chains.